core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -1335,7 +1335,9 @@ private LockWrapper exclusiveCleanDb(Connection c) throws SQLException {
   /**
    * Test case for
    * <a href="https://issues.apache.org/jira/browse/CALCITE-4619">[CALCITE-4619]
-   * "Full join" generates an incorrect execution plan under mysql</a>. */
```

Comment:
I feel a separate test case would be cleaner, `testFullJoinSupportedDialect` or similar for those who support it, the existing (untouched?) for those who don't

Reply:
Thanks for the suggestion and agree with it，had add a new test case for the issue.